dq-server/src/validation/input.js

```diff
@@ -156,6 +156,16 @@ const createGameSchema = Joi.object({
 	themes: Joi.array().items(Joi.objectId()).required()
 });
 
+const updateGameSchema = Joi.object({
+	name: Joi.string().required(),
+
+	players: Joi.array().items(Joi.objectId()).required(),
+
+	themes: Joi.array().items(Joi.objectId()).required(),
+	
+	otherwise: Joi.valid(null),
+});
+
```

Comment:
The front will send always the complete object with the attributes, name players and themes. 

So to only update themes, just change the themes and send the players as they are.

Reply:
It's ok, but I'd change it for the sake of consistency, because, if I recall correctly, other PUT endpoints in the app work as I said. You can leave it as it is.